src/elm/Pages/Deployments/View.elm

```diff
@@ -258,3 +274,37 @@ redeployLink org repo deployment =
         ]
         [ text "Redeploy"
         ]
+
+
+{-| pullBuildLinks : takes deployment and creates a list of links to every build in the builds field
+-}
+pullBuildLinks : Deployment -> List String
+pullBuildLinks deployment =
+    case deployment.builds of
+        Nothing ->
+            []
+
+        Just builds ->
+            List.map (String.dropRight 1) (List.map (String.dropLeft 1) (List.map Debug.toString (List.map .link builds)))
```

Comment:
`Debug.` shouldn't be included in code. Is that leftover from debugging? Should this just be `List.map .link builds`?

Reply:
yes it should be! thank you!